pandas/tests/test_lib.py

```diff
@@ -184,6 +185,52 @@ def test_get_reverse_indexer(self):
         self.assertTrue(np.array_equal(result, expected))
 
 
+class TestNullObj(tm.TestCase):
+
+    def test_basic(self):
+        arr = np.array([1, None, 'foo', -5.1, pd.NaT, np.nan])
+        expected = np.array([False, True, False, False, True, True])
+
+        result = lib.isnullobj(arr)
+        tm.assert_numpy_array_equal(result, expected)
+
+    def test_non_ndarray_inp(self):
+        ind = pd.Index([1, None, 'foo', -5.1, pd.NaT, np.nan])
```

Comment:
Not a very helpful comment tbh.  This a `lib` method, so I don't see why we don't put the tests here and instead go "hunting" for unit tests like this.